### PR TITLE
fix: Troubleshooter - Vertical scrolling not working on left panel or calendar #12751

### LINKED ISSUE #12751: [CAL-2796] Troubleshooter - Vertical scrolling not working on left panel or calendar
When the left panel is higher than the viewport, you are unable to scroll.

In addition to this, there is some weird behaviour causing it to show the content from the bottom up, rather than the top down. As a result, you can't access the back button

This scenario is most likely to occur if you have more than one calendar connected, however it of course depends on the viewport height.

Also, scrolling on the right panel does not appear to be working.

[https://github.com/calcom/cal.com/assets/4536123/48485486-7dc2-47fa-8b4c-ea7e89bb2db7](https://github.com/calcom/cal.com/assets/4536123/48485486-7dc2-47fa-8b4c-ea7e89bb2db7)

<sub>From [SyncLinear.com](https://synclinear.com) | [CAL-2796](https://linear.app/calcom/issue/CAL-2796/troubleshooter-vertical-scrolling-not-working-on-left-panel-or)</sub>

### PR DESCRIPTION
**Pull request by AsbarK**
## What does this PR do?

I have changed the css for the calender, So it is scrollable.(i have also changed the TroubleshooterSidebar.tsx according to prettier.)

Fixes #12751

https://github.com/calcom/cal.com/assets/106991686/66b247b1-0e75-4f99-b5f7-cd766aea6586

## Type of change

- Bug fix (non-breaking change which fixes an issue)

## How should this be tested?
u can visit http://localhost:3000/availability/troubleshoot?eventType and u  can see scrolable calender.

## Mandatory Tasks

- [x] Make sure you have self-reviewed the code. A decent size PR without self-review might be rejected.

### CODE REVIEW

File: packages/features/troubleshooter/components/TroubleshooterSidebar.tsx [hunk 1]
```diff
@@ -29,7 +29,7 @@ export const TroubleshooterSidebar = () => {
   const { t } = useLocale();
 
   return (
-    <div className="relative z-10 hidden w-full flex-col gap-6 h-screen overflow-y-auto py-6 pl-4 pr-6 sm:flex md:pl-0">
+    <div className="relative z-10 hidden h-screen w-full flex-col gap-6 overflow-y-auto py-6 pl-4 pr-6 sm:flex md:pl-0">
```
Comment (Udit-takkar) [inline packages/features/troubleshooter/components/TroubleshooterSidebar.tsx:32]: did you add anything here?

Comment (Udit-takkar) [reply packages/features/troubleshooter/components/TroubleshooterSidebar.tsx:32]: can you undo this

Comment (AsbarK) [reply packages/features/troubleshooter/components/TroubleshooterSidebar.tsx:32]: no i just changed it according to prettier.

Comment (keithwillcode) [reply packages/features/troubleshooter/components/TroubleshooterSidebar.tsx:32]: The classes were reshuffled based on prettier

Comment (AsbarK) [reply packages/features/troubleshooter/components/TroubleshooterSidebar.tsx:32]: > can you undo this? I don't see any change

yes

**Review by sean-brydon — CHANGES_REQUESTED** [review]
From what I can see - the scroll works but for some reason it recreates the same issue we had on booker where the busy slots (shaded areas) don’t scroll. If you zoom out you can see that they exist but for some reason here on scroll you can’t see them.

I’m absolutely baffled by this issue spent a good few hours looking at this

**Comment by AsbarK** [comment]
> From what I can see - the scroll works but for some reason it recreates the same issue we had on booker where the busy slots (shaded areas) don’t scroll. If you zoom out you can see that they exist but for some reason here on scroll you can’t see them.
> 
> I’m absolutely baffled by this issue spent a good few hours looking at this

i will try to fix that.

**Comment by sean-brydon** [comment]
> > From what I can see - the scroll works but for some reason it recreates the same issue we had on booker where the busy slots (shaded areas) don’t scroll. If you zoom out you can see that they exist but for some reason here on scroll you can’t see them.
> > I’m absolutely baffled by this issue spent a good few hours looking at this
> 
> i will try to fix that.

If you take a look at https://i.cal.com/sean/cal?layout=week_view you will see a working example of the expected behaviour

**Pull request closed without merging**